spark-tpcds-benchmark-runner/src/main/java/com/palantir/spark/benchmark/metrics/BenchmarkMetrics.java

```diff
@@ -73,7 +73,9 @@ public BenchmarkMetrics(
                         JacksonSerializer.create(BenchmarkMetric.class))
                 .createOrOpen();
         if (!this.metricsBuffer.isEmpty()) {
-            log.warn("Found unflushed metrics in the buffer; attempting to flush");
+            log.warn(
+                    "Found unflushed {} metrics in the buffer; attempting to flush",
+                    SafeArg.of("numMetrics", metricsBuffer.size()));
             flushMetrics();
```

Comment:
could we make the flush metrics call not nuke the db for now until we figure out the issue? maybe move it to a new hashmap key or something?